app/utils/api_queries_bag.py

```diff
@@ -20,6 +20,25 @@ def do_bag_search_nummeraanduiding_id_by_bag_id(bag_id):
     return address_search.json()
 
 
+@retry(stop=stop_after_attempt(3), after=after_log(logger, logging.ERROR))
+def do_bag_search_nummeraanduiding_id_by_address(address):
+    """
+    Search BAG nummeraanduiding_id by using an address
+    """
+    params = {"postcode": address.postal_code, "huisnummer": address.number}
+    if address.suffix:
+        params.huisnummertoevoeging = address.suffix
+    if address.suffix_letter:
+        params.huisletter = address.suffix_letter
```

Comment:
`params` is een type `dict`. Daar kun je niet via de dot notatie props aan toe voegen of wijzigen.